tests/Features/Methods/ModelExtension.php

```diff
@@ -141,6 +141,11 @@ public function testUpdateOrCreate() : User
         return User::updateOrCreate([]);
     }
 
+    public function testfirstOrCreate() : User
```

Comment:
Can you change this to `testFirstOrCreate` ? To follow the same style as others.

Reply:
ok, sorry newbie for github.

Reply:
is now ok?

Reply:
No, still the same.

First update your fork with the latest changes of the `master` branch.

Reply:
If you have `upstream` remote defined in your fork, you can do `git pull upstream master` to update your fork.

Reply:
ok, let me try.